pkgs/top-level/python-packages.nix

```diff
@@ -912,6 +912,18 @@ in {
 
   allpairspy = callPackage ../development/python-modules/allpairspy { };
 
+  inherit (callPackage ../development/python-modules/ansible { })
+    ansible
+    ansible_2_5
+    ansible_2_6
+    ansible_2_7;
```

Comment:
What's the argument for having different versions available? We usually prefer not do that for Python packages.

Reply:
No reason. I would prefer a single version too. The issue is that `all-packages.nix` needs to have multiple versions of ansible. What would you suggest?

Reply:
How about using `overridePythonAttrs` in `all-packages.nix`?

Reply:
I added a fix that I would like to see what you think. `ansible` is only exposed in pythonPackages and overridePythonAttrs is used to expose multiple ansible versions as asked. I did the overrides in a seperate file. pkgs/tools/admin/ansible/default.nix

Reply:
Resolved.